observability/logging/logging-6.0/log6x-opentelemetry-data-model.adoc

```diff
@@ -0,0 +1,381 @@
+* `structured metadata`: 
+** Allows for detailed filtering and storage of key-value pairs.
+** Enables users to use direct labels for streamlined queries without requiring JSON parsing.
+
+With OTLP, users can filter queries directly by labels rather than using JSON parsing, improving the speed and efficiency of queries.
+
+[cols="1,1,1,1,1", options="header"]
+|===
+|Name |Location |Applicable Sources |Storage (LokiStack) |Comment
+
+|`log_source`
+|resource
+|all
+|required stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `openshift.log.source`
+
+|`log_type`
+|resource
+|all
+|required stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `openshift.log.type`
+
+|`kubernetes.container_name`
+|resource
+|container
+|stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `k8s.container.name`
+
+|`kubernetes.host`
+|resource
+|all
+|stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `k8s.node.name`
+
+|`kubernetes.namespace_name`
+|resource
+|container
+|required stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `k8s.namespace.name`
+
+|`kubernetes.pod_name`
+|resource
+|container
+|stream label
+|*(DEPRECATED)* Compatibility attribute, contains same information as `k8s.pod.name`
+
+|`openshift.cluster_id`
+|resource
+|all
+|
+|*(DEPRECATED)* Compatibility attribute, contains same information as `openshift.cluster.uid`
+
+|`level`
+|log
+|container, journal
+|
+|*(DEPRECATED)* Compatibility attribute, contains same information as `severityText`
+
+|`openshift.cluster.uid`
+|resource
+|all
+|required stream label
+|
+
+|`openshift.log.source`
+|resource
+|all
+|required stream label
+|
+
+|`openshift.log.type`
+|resource
+|all
+|required stream label
+|
+
+|`openshift.labels.*`
+|resource
+|all
+|structured metadata
+|
+
+|`k8s.node.name`
+|resource
+|all
+|stream label
+|
+
+|`k8s.namespace.name`
+|resource
+|container
+|required stream label
+|
+
+|`k8s.container.name`
+|resource
+|container
+|stream label
+|
+
+|`k8s.pod.labels.*`
+|resource
+|container
+|structured metadata
+|
+
+|`k8s.pod.name`
+|resource
+|container
+|stream label
+|
+
+|`k8s.pod.uid`
+|resource
+|container
+|structured metadata
+|
+
+|`k8s.cronjob.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.daemonset.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.deployment.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.job.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.replicaset.name`
+|resource
+|container
+|structured metadata
+|Conditionally forwarded based on creator of Pod
+
+|`k8s.statefulset.name`
+|resource
+|container
+|stream label
+|Conditionally forwarded based on creator of Pod
```

Comment:
For the "Pod" in all of these, lowercase to "pod" per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) about general references to API objects.